src/navigation/MainNavigator.tsx

```diff
@@ -14,6 +15,11 @@ export const MainNavigator = () => {
           component={Connections}
           options={{headerShown: false}}
         />
+        <Stack.Screen
+          name={'BookingScreen'}
+          component={() => <Booking />}
```

Comment:
React warning says that the arrow function here can cause performace issues. You can simply pass the Booking component just like above at the Connections screen.